components/camel-jbpm/pom.xml

```diff
@@ -156,11 +156,31 @@
         </dependency>
     </dependencies>
     <build>
+        <sourceDirectory>${project.build.directory}/generated-sources/java</sourceDirectory>
         <resources>
             <resource>
                 <directory>src/main/resources</directory>
                 <filtering>true</filtering>
             </resource>
         </resources>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>templating-maven-plugin</artifactId>
```

Comment:
It would be nice to store templating-maven-plugin-version inside the parent/pom.xml for common update support
Otherwise, LGTM